crowdsec-docs/docs/profiles/cti_profile.md

```diff
@@ -0,0 +1,64 @@
+---
+id: cti_profile
+title: CrowdSec CTI
+sidebar_position: 2
+---
+
+Here is an example of a profile that uses the CTI module to make decisions based on the background noise score of an IP address.
```

Comment:
I think it's a great idea. We might even add an example that adds a notification based on either BN score or known fp ?